src/dattobd.c

```diff
@@ -217,10 +217,10 @@ static struct block_device *blkdev_get_by_path(const char *path, fmode_t mode, v
 typedef enum req_op {
 	REQ_OP_READ,
 	REQ_OP_WRITE,
-	REQ_OP_DISCARD,         /* request to discard sectors */
-	REQ_OP_SECURE_ERASE,    /* request to securely erase sectors */
-	REQ_OP_WRITE_SAME,      /* write same block many times */
-	REQ_OP_FLUSH,           /* request for cache flush */
+	REQ_OP_DISCARD,		 /* request to discard sectors */
```

Comment:
i realize this wasn't you, but can you remove the extra spaces here so everything lines up?